### ISSUE TITLE
Feature request: adapter for icecream (distributed computing cluster)

### ISSUE THREAD

**Issue by TimonTerBraak** [windelbouwman/lognplot#32]
Distributed compiler with a central scheduler to share build load:
https://github.com/icecc/icecream

icecream comes with an 'icecream monitor' to visualize the active workload and distribution among the connected clients. It would be nice to visualize that in lognplot as well. The monitor works by listening on a UDP port. I guess it would be doable to translate the info to lognplot.

**Comment by windelbouwman** [comment]
Agreed that this appears do-able as a python script. This would make for a great first issue!
